jobs/schedulers.go

```diff
@@ -86,11 +86,12 @@ func (schedulers *Schedulers) Start() *Schedulers {
 			}
 
 			for {
+				timer := time.NewTimer(1 * time.Minute)
 				select {
 				case <-schedulers.stop:
 					mlog.Debug("Schedulers received stop signal.")
 					return
```

Comment:
Should we also add a `timer.Stop()` here to be consistent?  
Another option would be to allocate the timer only once (outside the loop) `Stop`  and `Reset` as needed and use `defer` for final cleanup.

Reply:
Idea with `Reset` is nice, but requires some refactoring IMO to be readable. It could be another PR, this for loop looks already very big and should be moved into function or something :)
Just stopped timer on exit for now.